offset_monitor/offset_monitor.go

```diff
@@ -0,0 +1,72 @@
+package offset_monitor
+
+import (
+	"context"
+	"github.com/Shopify/sarama"
+	"github.com/go-dmux/core"
+	consumergroup "github.com/go-dmux/kafka/consumer-group"
+	"github.com/go-dmux/metrics"
+	"strconv"
+	"time"
+)
+
+type OffMonitor struct {
+	SourceSinkMonitorEnabled       bool          `json:"source_sink_monitor_enabled"`
+	ProducerConsumerMonitorEnabled bool          `json:"producer_consumer_monitor_enabled"`
+	OffPollingInterval             core.Duration `json:"offset_polling_interval"`
+}
```

Comment:
let's keep config and handler to execute logic separate, else caller can set configs at runtime and execution can get impacted